auth-web/src/components/auth/staff/admin/ResubmitRequestDialog.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    persistent
+    :attach="attach"
+    content-class="resubmit-request-dialog"
+  >
+    <v-card flat id="resubmit-request-container">
```

Comment:
This id is not needed as the dialog itself can be selected with `.resubmit-request-dialog`.